Shared/Samples/Apply mosaic rule to rasters/ApplyMosaicRuleToRastersView.swift

```diff
@@ -0,0 +1,211 @@
+                        currentDrawStatus = drawStatus
+                    }
+                }
+                .overlay(alignment: .center) {
+                    if currentDrawStatus == .inProgress {
+                        ProgressView("Loading...")
+                            .padding()
+                            .background(.ultraThickMaterial)
+                            .cornerRadius(10)
+                            .shadow(radius: 50)
+                    }
+                }
+                .task(id: model.ruleSelection) {
+                    guard let rasterLayer = model.map.operationalLayers.first as? RasterLayer else {
+                        return
+                    }
+                    do {
+                        // Downloads raster from online service
+                        try await rasterLayer.load()
+                        await mapProxy.setViewpoint(
+                            Viewpoint(
+                                center: model.getCenter(),
+                                scale: 25000.0
+                            )
+                        )
+                    } catch {
+                        self.error = error
+                    }
+                }
+                .toolbar {
+                    ToolbarItemGroup(placement: .bottomBar) {
+                        Picker("Mosiac Rules", selection: $model.ruleSelection) {
+                            ForEach(RuleSelection.allCases, id: \.self) { rule in
+                                Text(rule.label)
+                            }
+                        }
+                        .onChange(of: model.ruleSelection) { ruleSelection in
+                            Task {
+                                model.updateMosiacRule(with: ruleSelection.rule)
+                                await mapProxy.setViewpoint(
+                                    Viewpoint(
+                                        center: model.getCenter(),
+                                        scale: 25000.0
+                                    )
+                                )
+                            }
+                        }
+                        .pickerStyle(.automatic)
+                    }
+                }
+        }
+        .errorAlert(presentingError: $error)
+    }
+}
+
+private enum RuleSelection: CaseIterable, Equatable {
+    static var allCases: [RuleSelection] {
+        return [.objectID, .northWest, .center, .byAttribute, .lockRaster]
+    }
+    
+    case objectID, northWest, center, byAttribute, lockRaster
+    
+    @available(*, unavailable)
+    case all
+    
+    var label: String {
+        switch self {
+        case .objectID:
+            return "Object ID"
+        case .northWest:
+            return "North West"
+        case .center:
+            return "Center"
+        case .byAttribute:
+            return "By Attribute"
+        case .lockRaster:
+            return "Lock Raster"
+        }
+    }
```

Comment:
```suggestion
    var label: String {
        switch self {
        case .objectID: "Object ID"
        case .northWest: "North West"
        case .center: "Center"
        case .byAttribute: "By Attribute"
        case .lockRaster: "Lock Raster"
        }
    }
```

Not a big deal but this you can make this a little simpler. Also, it might be go to have a doc comment for this property.